src/newcharacter.cpp

```diff
@@ -2428,6 +2429,10 @@ void save_template( player *u )
     if( name.length() == 0 ) {
         return;
     }
+    if( name.find( '/' ) != std::string::npos || name.find( '\\' ) != std::string::npos ) {
```

Comment:
You can use [`std::string::find_first_of`](http://en.cppreference.com/w/cpp/string/basic_string/find_first_of) to check for several characters in one function call.